api/src/swagger.config.ts

```diff
@@ -29,7 +28,7 @@ export const swaggerDocumentBuilder = new DocumentBuilder()
   .setTitle("AdamRMS")
   .setDescription(description)
   .setVersion("v2")
-  .setContact("AdamRMS", "", "")
+  .setContact("AdamRMS", "https://adam-rms.com/", "contact@bstudios.com")
```

Comment:
Are there other references to contact@bstudios.com that you've seen on the web? If so we need to pull them as that domain isn't registered

Reply:
Resolved this in the next commit. Misremembered the name of the domain.